src/praisonai/praisonai/scheduler/_base_scheduler.py

```
@@ -229,6 +229,28 @@ class _BaseAgentScheduler:
+    def _should_suppress_delivery(self, text: str) -> bool:
+        """Return True when a run's whole output is an exact silence marker.
+
+        Honours the core intentional-silence contract (NO_REPLY / [SILENT] /
+        SILENT) on the unattended path so both sync and async schedulers stay
+        quiet instead of delivering the literal marker. Prose that merely
+        mentions the token is unaffected (exact-match).
+        """
+        try:
+            from praisonaiagents.bots.silence import is_intentional_silence_response
+            return is_intentional_silence_response(text)
+        except Exception:  # pragma: no cover - core primitive always present
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)